tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -98,25 +112,30 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
 			10	Deleted the SPBM polices created in step 1
 	*/
 	ginkgo.It("[csi-block-vanilla][csi-block-vanilla-parallelized][csi-guest][csi-supervisor]"+
-		" Verify Thin, EZT, LZT volume creation via SPBM policies", func() {
+		"[csi-wcp-vsan-direct] Verify Thin, EZT, LZT volume creation via SPBM policies", func() {
 
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
-
-		sharedvmfsURL := os.Getenv(envSharedVMFSDatastoreURL)
-		if sharedvmfsURL == "" {
-			ginkgo.Skip(fmt.Sprintf("Env %v is missing", envSharedVMFSDatastoreURL))
-		}
+		sharedvmfsURL, vsanDDatstoreURL := "", ""
+		var allocationTypes []string
 
 		scParameters := make(map[string]string)
 		policyNames := []string{}
 		pvcs := []*v1.PersistentVolumeClaim{}
 		pvclaims2d := [][]*v1.PersistentVolumeClaim{}
 		scs := []*storagev1.StorageClass{}
 
-		sharedvmfsURL = os.Getenv(envSharedVMFSDatastoreURL)
-		if sharedvmfsURL == "" {
-			ginkgo.Skip(fmt.Sprintf("Env %v is missing", envSharedVMFSDatastoreURL))
+		if wcpVsanDirectCluster {
```

Comment:
also `supervisorCluster` has to be set as well

Reply:
Done